resilience4j-framework-common/src/main/java/io/github/resilience4j/common/circuitbreaker/monitoring/endpoint/CircuitBreakerHystrixStreamEventsDTO.java

```diff
@@ -0,0 +1,75 @@
+package io.github.resilience4j.common.circuitbreaker.monitoring.endpoint;
+
+import io.github.resilience4j.circuitbreaker.CircuitBreaker;
+import io.github.resilience4j.circuitbreaker.CircuitBreakerConfig;
+import io.github.resilience4j.circuitbreaker.event.CircuitBreakerEvent;
+
+/**
+ * @author vijayram
+ */
+
+/**
+ * This class is used to emit available data as SSE so that resilence4j SSE can be bridged
+ * to suppose legacy hystrix eco tools such as hystrix dashboard , turbine and vizceral.
```

Comment:
@vijaycse please update the description to not mention `eco` :D

Reply:
I meant hystrix ecosystem of tools such as turbine. will update the docs

Reply:
Updated the document from eco to ecosystem